src/Generic/components/Address.tsx

```diff
@@ -94,7 +96,7 @@ interface AddressProps {
 }
 
 // tslint:disable-next-line no-shadowed-variable
-export const Address = React.memo(function Address(props: AddressProps) {
+const BasicAddress = React.memo(function BasicAddress(props: BasicAddressProps) {
```

Comment:
The name `BasicAddress` isn't great as it raises more questions than it answers. Tricky to find a better one, but how about `AddressContent`?

Reply:
I agree that the name is bad. `AddressContent` sounds reasonable 👍